data/posts/about.markdown

```diff
@@ -1,15 +1,15 @@
 
 # Jak jsme do toho praštili?
 
-Kromě jiného nás spojuje vášeň pro cestování. Naše dovolené byly často milníky v našem vztahu, třeba v Londýně 2014 jsme se rozhodli, že spolu začneme bydlet. Každé naše výročí pro sebe organizujeme nějaký výlet, kde ten druhý do poslední chvíle neví, kam se jede. Střídáme se v tom, kdo organizuje, tento rok to bylo na Davidovi. Bára musela nejdřív vyřešit programovací úkol a dozvěděla se souřadnice. Na souřadnicích vykopala krabičku s letenkou na Island hned druhý den. V Rejkjavíku jsme si [dali výborný Plokkfiskur](https://goo.gl/maps/Z9RjGYCzuos) a probírali u toho, že už je čas povýšit náš vztah na vyšší level a přemýšlet o rodině. Doma jsme si ještě nějaké věci sepsali, abychom věděli do čeho přesně jdeme. Shodou okolností jsme měsíc potom jeli do Vietnamu, jedna z dalších nezapomenutelných dovolených. David si naplánoval, že to bude ideální místo pro žádost o ruku. Bára nic netušila, "protože jsme byli domluvení". Příležitost přišla na ostrově [Phu Quoc](https://goo.gl/maps/mEtzXG7iUPs) na pláži [Ong Lang Beach](https://goo.gl/maps/918cM1WPFG72). Šli jsme tam docela dlouho, ale odměnou byla divoká pláž bez turistů jenom pro nás. Vzali jsme si s sebou zásoby a vyrobili si koktejly z místního ovoce. David požádal Báru ať zavře oči, Bára dostala prstýnek a řekla ANO!
+Kromě jiného nás spojuje vášeň pro cestování. Naše dovolené byly často milníky v našem vztahu, třeba v Londýně v létě 2014 (jen 5 měsíců po našem seznámení) jsme se rozhodli, že spolu začneme bydlet. Každé naše výročí pro sebe organizujeme nějaký výlet, kde ten druhý do poslední chvíle neví, kam se jede. Střídáme se v tom, kdo organizuje, tento rok to zrovna vyšlo na Davida. Bára musela nejdřív vyřešit programovací úkol, čímž získala souřadnice. Na nich vykopala krabičku s letenkou na Island hned druhý den. V Rejkjavíku jsme si [dali výborný Plokkfiskur](https://goo.gl/maps/Z9RjGYCzuos) a probírali u toho, že už je čas povýšit náš vztah na vyšší level a přemýšlet o rodině. Shodou okolností jsme měsíc poté jeli do Vietnamu, což byla jedna z dalších nezapomenutelných dovolených. David si naplánoval, že to bude ideální místo pro žádost o ruku. Bára nic netušila, „protože jsme byli domluvení“. Příležitost přišla na ostrově [Phu Quoc](https://goo.gl/maps/mEtzXG7iUPs) na pláži [Ong Lang Beach](https://goo.gl/maps/918cM1WPFG72). Šli jsme tam docela dlouho, ale odměnou byla divoká pláž bez turistů jenom pro nás. Vzali jsme si s sebou zásoby a vyrobili si koktejly z místního ovoce. David požádal Báru, ať zavře oči, Bára dostala prstýnek a řekla ANO! _(Ve skutečnosti jsem řekla „tak jo“, ale to by tady neznělo tak romanticky :D)_
```

Comment:
Klidně bych to ANO přepsal rovnou na "tak teda jo" :)

Reply:
Já ne, ať je to vtipnější.

Reply:
OK, nenecháme tam třetí osobu? "řekla" místo "jsem řekla"